### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by ghost**
Added `autocomplete="off"` to input.
Changed event listener to `input` from `keypress`.
Changed position to absolute on paper-menu.

### CODE REVIEW

File: src/ralph/admin/static/elements/auto-complete.html [hunk 1]
```diff
@@ -151,9 +158,12 @@
           value: function(){return [];}
         }
       },
-      handleTyping: function(evt, obj) {
-        console.log('handleTyping', evt, obj);
-        if (evt.target.value === '' || evt.target.value.length < this.fireThreshold) return false;
+      sendQueryRequest: function(evt, obj) {
+        console.log('sendQueryRequest', evt, obj);
```
Comment (szok) [inline src/ralph/admin/static/elements/auto-complete.html:162]: remove :)

Comment (mkurek) [inline src/ralph/admin/static/elements/auto-complete.html:162]: i thought about removing it in some gulp action. But do we have versioned static html files/templates? :)

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**